client/lib/utils/coerce-values.js

```diff
@@ -1,3 +1,5 @@
+let coerceFormValues;
```

Comment:
Before, when validating a field of type `array`, `coerceFormValues` was called recursively for each of its elements. The problem is, `coerceFormValues` only operates on fields of type `object` (like the entire `schema`, or the `box` definition). Since the Rates field definition is an `array` of plain `string`s, this code was blowing up.

I refactored it using my best judgement without completely flipping everything upside-down (I didn't expect the ESLint failure here). So, `coerceFormValues` works on `object`s, and `coerceValue` works on everything else (including `array`s). Do you have a better alternative?